test/Interop/Cxx/foreign-reference/inheritance.swift

```diff
@@ -0,0 +1,33 @@
+// REQUIRES: executable_test
+// RUN: %target-run-simple-swift(-cxx-interoperability-mode=default -Xfrontend -disable-availability-checking -I %S/Inputs)
+
+import StdlibUnittest
+import Inheritance
+
+// A function whose explicit type annotations specializes the cxxCast function.
+//
+// In Swift, the generic type parameters of cxxCast I and O should be respectively instantiated to SubT and BaseT.
+// However, since these are foreign reference types, this instantiates (and calls) cxxCast<SubT *, BaseT *> in C++.
+func Cast(_ s: SubT) -> BaseT {
```

Comment:
Minor: Swift functions are conventionally named starting from a lower case, e.g. `cast` in this case.